Update muon interface docs (#25781)
mantidproject/mantid: docs/source/interfaces/muon_fitting_tab.rst

.. _muon_fitting_tab-ref:

Fitting Tab
-----------

.. image::  ../images/muon_interface_tab_fitting.png
   :align: right
   :height: 500px

> The tools in this tab are built to assist in fitting functions to data from muon instruments.

Author: ```suggestion
The tools in this tab are built to assist in fitting functions to time data from muon instruments.
```

Author: ```suggestion
The tools in this tab are built to assist in fitting functions to time domain data from muon instruments.
```

> The tools in this tab are built to assist in fitting functions to data from muon instruments.

Reviewer: ```suggestion
This tab is for performing fits on the data.
```

> **Fit** Fits the functions in the function browser to the selected group(s)

Reviewer: ```suggestion
**Fit** Executes the chosen fit for the fitting functions to the selected data.
```

> **Single Fit** Choose this option to run the fitting tool for only a single group

Author: ```suggestion
**Single Fit** If selected the fitting tool will be for only a single group
```

Reviewer: ```suggestion
**Single Fit** This option performs a fit for a single data set.
```

> **Sequential Fit** Choose this option to run the fitting tool for several groups sequentially

Author: ```suggestion
**Sequential Fit** If selected the fitting tool will be run for several groups sequentially
```

Reviewer: ```suggestion
**Sequential Fit** This option will perform multiple fits sequentially, using the output of the previous fit as the starting guess. 
```

> **Simultaneous Fit** Choose this option to run the fitting tool for several groups simultaneously

Author: ```suggestion
**Simultaneous Fit** If selected the fitting tool will be run for several groups simultaneously
```

Reviewer: ```suggestion
**Simultaneous Fit** This option will perform a fit for multiple data sets, which may share parameters. 
```

> **Select data to fit** this opens up the dialog box to choose which of the groups in the currently loaded run(s) you

Author: ```suggestion
**Select data to fit** This opens up the dialog box to choose which of the groups in the currently loaded run(s) to
```

Reviewer: ```suggestion
**Select data to fit** This launches a dialog box for selecting the data to be used in the fit.
```

> want to fit to.

Author: ```suggestion
fit to.
```

Reviewer: ```suggestion
```

> **Display parameters for** When fitting data to multiple workspaces use this to switch between different selected

Author: ```suggestion
**Display parameters for** When fitting data to multiple workspaces this can be used to switch between different selected
```

Reviewer: ```suggestion

```

> workspaces and manipulate initial values for fit functions separately.

Reviewer: ```suggestion
```

Fit function browser
^^^^^^^^^^^^^^^^^^^^

> Right clicking in the central box will bring up a context menu allowing you to add or remove functions or backgrounds.

Author: ```suggestion
Right clicking in the central box will bring up a context menu allow for adding or removing functions or backgrounds.
```

Reviewer: ```suggestion
Right clicking in the central box will bring up a context menu for adding or removing functions.
```

> Once functions or backgrounds are added to the browser you can use to modify the initial guess for a varabile and set it

Author: ```suggestion
Once functions or backgrounds are added to the browser initial guesses for variables can be modified or set
```

Reviewer: ```suggestion
Once functions have been added to the browser the user can adjust the values of the parameters. 
```

> to be shared between all other functions in this fit with the global check box.

Reviewer: ```suggestion
If the parameter is to be the same for all of the data sets, the global check box should be ticked.
```

All selected workspaces will share the same composite fitting function but can have different variables.

Fit properties browser
^^^^^^^^^^^^^^^^^^^^^^

> **Time Start/End** The values here set the boundaries that the fitting function will be calculated between.

Reviewer: ```suggestion
**Time Start/End** Defines the boundary values for the fit calculation.
```

> **Minimizer** Choose the technique to minimize the fitness function.

Author: ```suggestion
**Minimizer** this can be used to select the technique used to minimize the fitness function.
```

Reviewer: ```suggestion
**Minimizer** Choose the minimisation method for the fit.
```

> **Fit To Raw Data** If this is checked it will use the raw data from the instrument with its default bins, if it is

Reviewer: ```suggestion
**Fit To Raw Data** If this is checked it will use the raw data for the fit.
```

> unchecked it will use data rebinned using the specifications from the home tab.

Reviewer: ```suggestion
If it is unchecked it will use the rebinned data as specified on the home tab.
```

> **Evaluate Function As** Using ths you can choose weather the fitting function will treat the function as a histogram or

Author: ```suggestion
**Evaluate Function As** This allows for the function to be treated as a histogram or
```

Reviewer: ```suggestion
**Evaluate Function As** Select if to fit to histogram or point data.
```

> a CenterPoint.

Reviewer: ```suggestion
```
